lib/rules/disallow-space-after-object-keys.js

```diff
@@ -134,7 +134,7 @@ module.exports.prototype = {
             var maxKeyEndPos = 0;
             var tokens = [];
             node.properties.forEach(function(property) {
-                if (property.shorthand || property.kind !== 'init') {
+                if (property.shorthand || property.kind !== 'init' || property._babelType === 'SpreadProperty') {
```

Comment:
Ah, it's es7! I guess the complications with supporting es7 syntax is already rearing its head. I understand now, you can't/shouldn't modify the tree with an unsupported node type, so `_babelType` was created to hold artificial types. 

What concerns me is the maintenance of rules that depend on a private property (`_babelType` ) that was made up by an ast transformation lib. IF this feature gets standardized by EStree, then we'll have to wait for Esprima or Acorn to implement it, then update our rule to stop using `_babelType`.

* Making `_babelType` a public property doesn't really solve the issue (though it makes a statement that the artificial node type is part of babel-jscs' public api)

Would love an opinion from @mdevils, @markelog, or @mikesherov on suggestions for moving forward with es7 features.